### PR TITLE
Adding Docker automation scripts

### PR DESCRIPTION
**Pull request by fabiononato**
Automation scripts for Building and Running a Dockerized version of the notebooks were included in the base repo. 

run.sh runs the image and provide a Jupyter Notebook Server, that can be reached from localhost:8888.

The image is built on top of tf2-gpu, but can be executed on CPU machines as well.

### CODE REVIEW

**Comment by twiecki** [comment]
Thanks for working on this @fabiononato!

**Comment by twiecki** [comment]
@fabiononato What's missing from this? Still shows up as draft.

**Comment by fabiononato** [comment]
@twiecki ... needed merging from upstream for your new files. All good now! Thanks

File: README.md [hunk 1]
```diff
@@ -1,8 +1,29 @@
 # covid19
+<<<<<<< HEAD
```
Comment (twiecki) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+Copy and past the link in your browser to access the Server.
+=======
 Analyses about the COVID-19 virus.
 
 Interactive notebooks on mybinder:
 
 https://mybinder.org/v2/gh/twiecki/covid19/master?filepath=covid19_growth.ipynb
 
 
+>>>>>>> upstream/master
```
Comment (twiecki) [inline README.md]:
```suggestion
```

**Comment by twiecki** [comment]
Thanks @fabiononato!

**Comment by fabiononato** [comment]
@twiecki ... might need to update requirements a couple more, still missing this `altair` & `arviz`. I'm not familiar with this import on the new notebook: `sunode.wrappers.as_theano`

Will probably need to work on adding vega3 for altair to work in docker. Can continue on later in the week!

Thanks

**Pull request merged**